src/ReleaseHistory.md

```diff
@@ -5,6 +5,7 @@
 * BREAKING: `AnalyzeCommandBase` previously persisted all scan target artifacts to SARIF logs rather than only persisting artifacts referenced by an analysis result, when an option to persist hashes, text file or binary information was set. `MultithreadedAnalyzeCommandBase` previously persisted all scan targets artifacts to SARIF logs in cases when hash insertion was eenabled rather than only persisting artifacts referenced by an analysis result. [#2433](https://github.com/microsoft/sarif-sdk/pull/2433)
 * BUGFIX: Adjust Json Serialization field order for ReportingDescriptor and skip emit empty AutomationDetails node. [#2420](https://github.com/microsoft/sarif-sdk/pull/2420)
 * BREAKING: Fix `InvalidOperationException` when using PropertiesDictionary in a multithreaded application, and remove `[Serializable]` from it. Now use of BinaryFormatter on it will result in `SerializationException`: Type `PropertiesDictionary` is not marked as serializable. [#2415](https://github.com/microsoft/sarif-sdk/pull/2415)
+* BUGFIX: Notifications should emit an artifact if artifactLocation is not null. [#2437](https://github.com/microsoft/sarif-sdk/pull/2437)
```

Comment:
>Notifications should emit an artifact if artifactLocation is not null [](http://example.com/codeflow?start=10&length=69)

You didn't document what type's behavior changed, it's SarifLogger. You need this information for customers to assess whether the change impacts them.

`SarifLogger` now emits an artifacts table entry is `artifactLocation` is not null for tool configuration and tool execution notifications.

Reply:
Just updated!
thanks for the suggestion!